src/main/java/com/github/kmu_shell_we/domain/mission/repository/SeasonMissionRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.github.kmu_shell_we.domain.mission.repository;
+
+import com.github.kmu_shell_we.domain.mission.entity.Mission;
+import com.github.kmu_shell_we.domain.mission.entity.SeasonMission;
+import com.github.kmu_shell_we.domain.season.entity.Season;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface SeasonMissionRepository extends JpaRepository<SeasonMission, UUID> {
+
+    SeasonMission findBySeasonAndMission(Season season, Mission mission);
```

Comment:
항상 NPE방지를 위해 Optional로 감싸서 안전하게 사용해주세요.


```suggestion
    Optional<SeasonMission> findBySeasonAndMission(Season season, Mission mission);
```